Makefile

```diff
@@ -35,5 +35,12 @@ kernel: $(KERNEL_CC_LIST) $(KERNEL_AS_LIST)
 run:
 	qemu-system-i386 -kernel exylus.bin
 
+iso: kernel
+	cp exylus.bin iso/boot/exylus.bin
+	grub-mkrescue -o exylus.iso iso/
+
+run-iso: iso
+	qemu-system-i386 -cdrom exylus.iso
+
 clean:
 	find -type f \( -name "*.o" -o -name "*.bin" \) -delete
```

Comment:
You should add a filter for `.iso` files here. Add a newline at the end of the file, I missed it somehow.